src/components/FeesPanel.tsx

```diff
@@ -15,11 +15,14 @@ import { gasUtils } from '@/utils';
 import { Box, Inline, Inset, Row, Rows, Text } from '@/design-system';
 import { IS_ANDROID, IS_TEST } from '@/env';
 import { isL2Chain } from '@/handlers/web3';
-import { ExplainSheetRouteParams, CurrentBaseFeeTypeKey, RootStackParamList, gasTrendToTrendType } from '@/navigation/types';
+import { CurrentBaseFeeTypeKey, ExplainSheetRouteParams, gasTrendToTrendType, RootStackParamList } from '@/navigation/types';
+import { ChainId } from '@/state/backendNetworks/types';
 import { useNavigation } from '@/navigation';
 const MAX_TEXT_WIDTH = 210;
 const { CUSTOM, GAS_TRENDS, NORMAL, URGENT } = gasUtils;
 
+const chainsThatIgnoreThePriorityFee = [ChainId.arbitrum, ChainId.arbitrumNova, ChainId.arbitrumSepolia];
+
```

Comment:
would love to see if we have this data somehow delivered via meteorology / backend